lib/gems/pending/util/miq_winrm.rb

```diff
@@ -10,6 +7,10 @@ def initialize
     @shell           = nil
     @elevated_runner = nil
     require 'uri'
+
+    # These are slow/bulky (15 MB+) requires so delay them until they're needed
+    require 'winrm'
+    require 'winrm-elevated'
```

Comment:
Can you upstream this change separately?

Reply:
yeah, I was playing with eager load and found/remembered this.

Reply:
For future me, I hope this whole file can be deleted because of https://github.com/ManageIQ/manageiq/pull/22279

The only callers of `MiqWinRM` in my bundle are this file and it's test and two files in smartstate.

```
% rg -M 300 "MiqWinRM" ./ `gem env home`
/Users/joerafaniello/.gem/ruby/2.7.6/bundler/gems/manageiq-gems-pending-d4ced36efe52/spec/util/miq_winrm_spec.rb
3:describe MiqWinRM do
8:    @winrm    = MiqWinRM.new

/Users/joerafaniello/.gem/ruby/2.7.6/bundler/gems/manageiq-gems-pending-d4ced36efe52/lib/gems/pending/util/miq_winrm.rb
4:class MiqWinRM

/Users/joerafaniello/.gem/ruby/2.7.6/gems/manageiq-smartstate-0.8.1/lib/Scvmm/miq_hyperv_disk.rb
23:    @winrm     = MiqWinRM.new

/Users/joerafaniello/.gem/ruby/2.7.6/gems/manageiq-smartstate-0.8.1/lib/Scvmm/miq_scvmm_vm_ssa_info.rb
13:    @winrm       = MiqWinRM.new
```

cc @agrare   🙏

Reply:
Ok, I removed the commit to delay load winrm as it's being deleted in https://github.com/ManageIQ/manageiq-gems-pending/pull/556

Reply:
Resolving